src/user/dto/user.dto.ts

```diff
@@ -44,7 +48,9 @@ export class CreateUserResponse {
 
 export class LoginDto {
   @ApiProperty()
-  @IsPhoneNumber()
+  @Length(10, 10)
+  @Matches(/^0([0-9]{9})$/, { message: 'the phone number is invalid' })
+  // @IsPhoneNumber()
```

Comment:
Please talk to the team again about handling international phone but this is quite urgent so i'll approve for now.

Reply:
kkkkkkkk